docs/translations/translations.json

```
@@ -295,6 +295,7 @@
     "/guides": "How To Guides",
     "/guides/typescript": "TypeScript",
     "/guides/interoperability": "Style Library Interoperability",
+    "/guides/styled-engine": "Styled Engine",
     "/guides/minimizing-bundle-size": "Minimizing Bundle Size",
```

Off-topic. @mbrookes Should we change the capitalization of the links to:

```suggestion
    "/guides/minimizing-bundle-size": "Minimizing bundle size",
```

?